site-generator/.gitignore

```diff
@@ -1 +1,2 @@
-result
\ No newline at end of file
+result
+/.vscode
```

Comment:
I personally commit `.vscode` to the repository so that I have the workspace settings in sync from wherever I work. Could you remove it from the PR?